services/transaction_body.proto

```diff
@@ -288,6 +290,16 @@ message TransactionBody {
      */
     TokenFeeScheduleUpdateTransactionBody token_fee_schedule_update = 45;
 
+    /**
+     * Pauses the Token
+     */
+    TokenPauseTransactionBody tokenPause = 46;
```

Comment:
@steven-sheehy would these have to be snake case as well ? Its not consistent
across the file

Reply:
I don't think you need to change fields you didn't add.